sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceAsyncClient.java

```diff
@@ -223,7 +223,7 @@ public TableAsyncClient getTableClient(String tableName) {
      * @param tableName The name of the table to delete.
      * @return A reactive result containing the HTTP response.
      * @throws IllegalArgumentException If {@code tableName} is {@code null} or empty.
-     * @throws TableServiceErrorException If no table with the provided name exists within the service.
+     * @throws TableServiceException If no table with the provided name exists within the service.
```

Comment:
I'm guessing the 404-swallowing will be in a separate PR?

Reply:
Correct

Reply:
Although I could add it on this one if you feel it makes sense.

Reply:
Nah - separate is good.